tests/test_control_bands.py

```diff
@@ -0,0 +1,181 @@
+from collections import defaultdict
+from unittest.mock import MagicMock
+from time import time
+from datetime import timedelta
+from statistics import mean, stdev
+from ..control_bands_block import ControlBands
+from nio.common.signal.base import Signal
+from nio.util.support.block_test_case import NIOBlockTestCase
+
+
```

Comment:
I would have named this file "test_control_bands_block.py". If we want to keep that consistent...